data/json/items/resources/tailoring.json

```
@@ -392,6 +393,37 @@
     "symbol": "=",
     "color": "dark_gray"
   },
+  {
+    "id": "sheet_kevlar_patchwork",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": { "str": "patchwork Kevlar sheet" },
+    "description": "A 12-inch by 24-inch sheet of fabric sewn together from Kevlar patches, suitable for making durable cut and stab resistant clothing.  This would be out of the question for ballistic Kevlar but the simpler, looser weave of this variety means it is fine to use if you lack proper material sheets.",
+    "weight": "26 g",
+    "volume": "51 ml",
+    "//": "values taken from Duponts chemical reference. 144 GSM",
+    "price": 900,
+    "price_postapoc": 50,
+    "material": [ "kevlar" ],
+    "looks_like": "sheet_kevlar",
+    "symbol": "=",
+    "color": "dark_gray"
+  },
+  {
+    "id": "kevlar_patch",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": { "str": "Kevlar patch", "str_pl": "Kevlar patches" },
+    "description": "A 6-inch square patch of loose-woven kevlar, suitable for repairing cut and stab resistant clothing or reassembling into sheet for sewing.",
```

```suggestion
    "description": "A 6-inch square patch of loose-woven Kevlar, suitable for repairing cut and stab resistant clothing or reassembling into sheet for sewing.",
```
*Angry DuPont noises*